doc/man3/SSL_get_verify_result.pod

```diff
@@ -2,25 +2,30 @@
 
 =head1 NAME
 
-SSL_get_verify_result - get result of peer certificate verification
+SSL_get_verify_result,
+SSL_SESSION_get_verify_result
+- get result of peer certificate verification
 
 =head1 SYNOPSIS
 
  #include <openssl/ssl.h>
 
  long SSL_get_verify_result(const SSL *ssl);
+ long SSL_SESSION_get_verify_result(const SSL_SESSION *ss);
```

Comment:
That shouldn't be here. Looks to have been a porting issue on my part.

Reply:
Ah... now I remember. This allows the client to validate the certificate result of the session before using it. If the X.509 validation result is not what is expected (i.e. not success), the client may choose to not use that session, and then instead create a new session, possibly with a different certificate. There is no other way to retrieve the X.509 validation state of a session.
This is critical for being able to resume a session that uses client certificate authentication.

Reply:
But I assume that the SSL_SESSION could be assigned via SSL_set_session(), and then the SSL's verify status checked at that point (since it's copied into the SSL). It would then need to be NULL'd if not wanted.
Which would also work with the suggestion of not having two-step process of retrieving the session, then assigning it.